sdk/test-utils/perfstress/GettingStarted.md

```diff
@@ -0,0 +1,263 @@
+# Writing Performance Tests
+
+## [Index](#index)
+
+- [Setting up the project](#Setting-up-the-project)
+  - [Track 2](#Setting-up-the-project)
+  - [Track 1](#For-perf-testing-track-1-version-of-the-same-package)
+- [Writing perf tests](#writing-perf-tests)
+  - [Entry Point](#Entry-point)
+  - [Base Class](#Base-Class)
+  - [Test File](#Test-file)
+  - [Custom Options](#custom-options)
+- [Executing the perf tests](#executing-the-perf-tests)
+  - [Command to run](#Command-to-run)
+  - [Adding Readme/Instructions](#Adding-Readme/Instructions)
+  - [Testing an older track 2 version](#Testing-an-older-track-2-version)
+
+## [Setting up the project](#Setting-up-the-project)
+
+To add perf tests for the `sdk/<service>/<service-sdk>` package, follow the steps below.
+
+1.  Create a new folder for the perf tests.
+
+    Path- `sdk/<service>/perf-tests/<service-sdk>`
```

Comment:
Sometimes having a filled in example is useful. I think:

From your template:
`sdk/<service>/<service-sdk>`

Then:
    <service> === "servicebus"
    <service-sdk> === "service-bus"

So now doing replacements: 
 Path- `sdk/<service>/perf-tests/<service-sdk>`

Then you want us to create this folder?

`sdk/servicebus/perf-tests/service-bus`

Is that correct (it doesn't seem right)

Reply:
Ah, maybe this is to avoid the nested projects thing you've mentioned before.

Reply:
> sdk/servicebus/perf-tests/service-bus

This is absolutely right. 😁